frontend/api/createAxiosInstance.ts

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+const createAxiosInstance = () => {
```

Comment:
Needs to be called at runtime as Nextjs tries to render `window.location.origin` in the Next server when there is no window